qa-tests-backend/src/main/groovy/util/NetworkGraphUtil.groovy

```diff
@@ -13,7 +13,9 @@ import services.NetworkGraphService
 @Slf4j
 class NetworkGraphUtil {
 
-    static final NETWORK_FLOW_UPDATE_CADENCE_IN_SECONDS = 30 // Network flow data is updated every 30 seconds
+    // more time is needed on few architectures
+    static final NETWORK_FLOW_UPDATE_CADENCE_IN_SECONDS =
+        ((System.getenv("REMOTE_CLUSTER_ARCH") == "x86_64" ) ? 30 : 120)
```

Comment:
`System.getenv("REMOTE_CLUSTER_ARCH")` is going to be "" by default. It is odd that this caused the GKE x86 arch test to fail. https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/5974/pull-ci-stackrox-stackrox-master-gke-qa-e2e-tests/1656024425654390784
I wonder is there something to how the collector/sensor/central deployment is configured that tunes these tests to work.

Reply:
Fixed and changed to `Env.REMOTE_CLUSTER_ARCH == "x86_64" ) ? 30 : 120`